ui/components/EventMembers/index.tsx

```diff
@@ -152,14 +144,26 @@ const EventMembers = ({ collection, currentUser }) => {
 
   const [, updateMember] = useMutation(UPDATE_MEMBER);
 
-  const [, deleteMember] = useMutation(DELETE_MEMBER);
+  const [{ fetching:deleteMemberLoading, data: deleteMemberResponse }, deleteMember] = useMutation(DELETE_MEMBER);
 
   const [inviteModalOpen, setInviteModalOpen] = useState(false);
 
   const isAdmin =
     currentUser?.currentOrgMember?.isAdmin ||
     currentUser?.currentCollMember?.isAdmin;
 
+  useEffect(() => {
+    const member = approvedMembers.find(_member => {
+      return _member.id === deleteMemberResponse?.deleteMember.id
+    });
+    
+    if (!deleteMemberLoading && (currentUser.id === member?.user?.id)) {
+      Router.reload();
+    }
+
+    console.log(deleteMemberResponse, currentUser);
+  }, [deleteMemberLoading]);
+
```

Comment:
If the purpose here is to remove the deleted member from the list, this should be done by updating the urql cache.

Reply:
added logic for clearing the cache now